browser/ui/brave_browser_window.h

```diff
@@ -44,7 +44,13 @@ class BraveBrowserWindow : public BrowserWindow {
   // the overall screen's height
   virtual gfx::Rect GetShieldsBubbleRect();
 
-  virtual std::map<int, std::vector<ui::Accelerator>> GetAcceleratedCommands();
+  virtual base::flat_map<int, std::vector<ui::Accelerator>>
+  GetAcceleratedCommands();
+  virtual void AddAcceleratorForCommand(int command_id,
+                                        ui::Accelerator accelerator);
+  virtual void RemoveAcceleratorForCommand(int command_id,
+                                           ui::Accelerator accelerator);
```

Comment:
nit: const & for accelerator

```suggestion
  virtual void AddAcceleratorForCommand(int command_id,
                                        const ui::Accelerator& accelerator);
  virtual void RemoveAcceleratorForCommand(int command_id,
                                           const ui::Accelerator& accelerator);
```